src/bio/bio_device.c

```diff
@@ -44,6 +44,7 @@ revive_dev(struct bio_xs_context *xs_ctxt, struct bio_bdev *d_bdev)
 	D_ASSERT(bbs->bb_state == BIO_BS_STATE_OUT);
 	D_ASSERT(owner_thread(bbs) != NULL);
 
+	d_bdev->bb_trigger_reint = 1;
```

Comment:
Yeah, the reason I didn't trigger reintegration for "device reviving" is that the "device reviving" is supposed being used by tests only, and we don't want the irrelevant tests being interfered by the premature reintegration feature.

But the recent changes for faulty WAL/Meta SSD changed faulty & reint reaction (where we bring up/down individual ds_pool_child), so the reint reaction can't be skipped anymore. Given the reintegration feature is much more stable now, I think we'd just enable reintegration for "device reviving" now.